cfgov/unprocessed/css/organisms/item-introduction.less

```diff
@@ -0,0 +1,46 @@
+/* topdoc
+  name: Item Introduction
+  family: cfgov-organisms
+  patterns:
+    - name: Default example
+      markup: |
+        <div class="o-item-introduction">
+            <div class="h4">…</div>
+            <h1>…</h1>
+            <p class="short-desc">…</p>
+            <p class="short-desc">
+              By <a href="#">AuthorA</a> -
+              <span><time>DEC 24, 2015</time></span>
+            </p>
+            <div>
+              [social media molecule]
+            </div>
+        </div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .o-item-introduction
+            .h4
+            h1
+            .short-desc
+            .short-desc
+              a
+              span
+                time
+            div
+              [social media molecule]
+  tags:
+    - cfgov-organisms
+*/
+
+.o-item-introduction {
+    .short-desc {
+        padding-bottom: unit( ( (@grid_gutter-width / 2) / @base-font-size-px), em)
```

Comment:
Removed them in https://github.com/cfpb/cfgov-refresh/commit/1e6e523e5b75f9125be2f6f45b97c68bfaa9c218 I guess there isn't an order of operations issue in `@grid_gutter-width / 2 / @base-font-size-px`